docs/user/instrument-android-crashes-example.md

```diff
@@ -0,0 +1,172 @@
+
+Now that a metric type has been chosen to represent the metric, the next step is creating the `metrics.yaml`.  Inside of the
+root application folder of the Android Studio project create a new file named `metrics.yaml`.  After adding the schema
+definition and event metric defninition, the `metrics.yaml` should look like this:
+
+```YAML
+# Required to indicate this is a `metrics.yaml` file
+$schema: moz://mozilla.org/schemas/glean/metrics/1-0-0
+
+crash:
+  exception:
+    type: event
+    description: |
+      Event to record crashes caused by unhandled exceptions
+    notification_emails:
+      - crashes@example.com
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1582479
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1582479
+    expires:
+      2021-01-01
+    send_in_pings:
+      - crash
+    extra_keys:
+      cause:
+        description: The cause of the crash
+      message:
+        description: The exception message
+```
+
+As a brief explanation, this creates a metric called `exception` within a metric category called `crash`.  There is a text
+`description` and the required `notification_emails`, `bugs`, `data_reviews`, and `expires` fields.  The `send_in_pings` field
+is important to note here that it has a value of `- crash`.  This means that the crash event metric will be sent via a custom
+ping named `crash` (which hasn't been created yet).  Finally, note the `extra_keys` field which has two keys defined, `cause`
+and `message`.  This allows for sending additional information along with the event to be associated with these keys.
+
+**Note:**  For Mozilla applications, a mandatory [data review](https://github.com/mozilla/data-review/blob/master/request.md) is required in order to collect information with the Glean SDK.
+
+### Add A Custom Ping
+
+Define the custom ping that will help control the upload scheduling by creating a `pings.yaml` file in the same directory as
+the `metrics.yaml` file.  For more information about adding custom pings, see the section on [custom pings](pings/custom.md).  
+The name of the ping will be `crash`, so the `pings.yaml` file should look like this:
+
+```YAML
+# Required to indicate this is a `pings.yaml` file
+$schema: moz://mozilla.org/schemas/glean/pings/1-0-0
+
+crash:
+  description: >
+    A ping to transport crash data
+  include_client_id: true
+  notification_emails:
+    - crash@example.com
+  bugs:
+    - https://bugzilla.mozilla.org/show_bug.cgi?id=1582479
+  data_reviews:
+    - https://bugzilla.mozilla.org/show_bug.cgi?id=1582479
+```
+
+Before the newly defined metric or ping can be used, the application must first be built.  This will cause the [glean_parser](https://github.com/mozilla/glean_parser/)
+to execute and generate the API files that represent the metric and ping that were newly defined.
+
+**NOTE:** If changes to the YAML files aren't showing up in the project, try running the clean task on the project before
```

Comment:
This document has "Important Note", "Note" and "NOTE". We should probably make this a consistent "Note"